File: Day7/Intersection Of Two Sorted Arrays/IntersectionOfTwoSortedArrays.py
def findArrayIntersection(arr: list, n: int, brr: list, m: int):
    # Write your code here
    # Return a list containing all the common elements in arr and brr.
    ans = []
    
    i = 0
    j = 0
    while i < n and j < m:
        if arr[i] > brr[j]:
            j += 1
        elif arr[i] < brr[j]:
            i += 1
        else:
            ans.append(arr[i])
            i += 1
            j += 1
    
    return ans
    pass

File: Day7/Intersection Of Two Sorted Arrays/IntersectionOfTwoSortedArrays.cs
public class Solution
{
    List<int> findArrayIntersection(List<int> arr1, int n, List<int> arr2, int m)
    {
        List<int> ans = new List<int>();
        
        int i = 0, j = 0;
        
        while(i < n && j < m)
        {
            if(arr1[i] < arr2[j])
            {
                i++;
            }
            else if(arr1[i] > arr2[j])
            {
                j++;
            }
            else
            {
                ans.Add(arr1[i]);
                i++;
                j++;
            }
        }
        
        return ans;
    }
}